packages/components/src/FormField/FormField.css

```diff
@@ -8,9 +8,9 @@
   --public-field--bottom-left-radius: var(--radius-base);
   --public-field--bottom-right-radius: var(--radius-base);
 
-  --field--placeholder-color: var(--color-greyBlue--light);
+  --field--placeholder-color: var(--color-base-blue--600);
```

Comment:
Considering moving these 3 component tokens up to the main semantic layer to make them more apparent for theming, but also very open to not doing that here